infra/prometheus/Makefile.toml

```diff
@@ -0,0 +1,29 @@
+extend = [
+  { path = "../fendermint/scripts/docker.toml" },
+]
+
+[tasks.prometheus-pull]
+script = "docker pull prom/prometheus"
```

Comment:
Let's just run this as a dependency of `prometheus-start`. (Although I think docker run will anyway pull from the Docker Registry automatically, so this might not even be needed at all?)

Reply:
Removed this task.